app/controllers/application_controller.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 class ApplicationController < ActionController::Base
+  include ActiveStorage::SetCurrent
```

Comment:
Since this is entirely untested, it would be nice to have some comments around this in the code.  If we remove it the tests all still pass, so someone might be tempted to revert this commit.